lib/dgram.js

```diff
@@ -495,6 +495,19 @@ Socket.prototype.setMulticastLoopback = function(arg) {
 };
 
 
+Socket.prototype.setMulticastInterface = function(interfaceAddress) {
+  this._healthCheck();
+
+  if (!interfaceAddress) {
```

Comment:
Instead of checking for falsey values, maybe this should check for strings explicitly.